app/logic/serviceLearningCoursesData.py

```diff
@@ -39,6 +42,17 @@ def withdrawProposal(courseID):
     """Withdraws proposal of ID passed in. Removes foreign keys first.
     Key Dependencies: QuestionNote, CourseQuestion, CourseParticipant,
     CourseInstructor, Note"""
+
+    # delete syllabus
+    try:
+        syllabi = AttachmentUpload.select().where(AttachmentUpload.course==courseID)
+        for syllabus in syllabi:
+            FileHandler(courseId = courseID).deleteFile(syllabus.id)
+
+    except DoesNotExist:
+        print("File does not exist")
```

Comment:
Need some more info here - there's no way for someone who sees this log while trying to solve a problem to know what file caused this error.